code_examples/sdk_examples/src/core_features/utils/getRandomCtype.ts

```diff
@@ -0,0 +1,14 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export function getRandomCType(): Kilt.ICType {
+  // Random factor ensures that each created CType is unique and does not already exist on chain.
+  const randomFactor = Kilt.Utils.UUID.generate()
+  return Kilt.CType.fromProperties(`CType ${randomFactor}`, {
```

Comment:
Do you always want a random cType? What is the benefit of this?

It feels like bad behaviour to teach people.

It would be more practical to teach people to check if a CType is on-chain and reuse it.

Reply:
Yes we shouldn't mention that it's a random ctype. We use this call since it's the easiest to construct. but we can use any. So i removed all mentions of the ctype in the visible code [babcf11](https://github.com/KILTprotocol/docs/pull/183/commits/babcf117a6541e2ec8fbe3e2e23248b254915005)